### ISSUE TITLE
Accessibility issues for the 'Facility operating statuses' block in CMS

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#22593]
## Describe the defect 
The `aria-describedby` attribute on the `<details>` tag of the 'Facility operating statuses' block is pointing to something with the ID `edit-field-facility-operating-status--description` but if I search for that ID I can't find it anywhere on the page, it doesn't exist. There is a description field in the details component, but that info can be accessed by opening the details and reading as usual, so [using aria is unnecessary](https://dsva.slack.com/archives/C8E985R32/p1761168017435559).

Some links to content type edit screens with this block:
-  [VAMC Operating Status]()
- 

## Issues & Recomendations
- [ ] Remove the `aria-describedby` on the `<details>` tag.

## AC / Expected behavior
- [ ] The 'Facility operating statuses' block no longer throws an error about the unlinked `aria-describedby`
- [ ] Accessibility review

## Screenshots
Screenshot of the Facility operating statuses field error from AMP plugin. The `aria-describedby` attribute is pointing to something with the ID `edit-field-facility-operating-status--description` but if I search for that ID I can't find it anywhere on the page, it doesn't exist. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/8dbd284c-8470-4ee7-b916-df4dd0da2ddd)

Screenshot of where the attribute is on the `<div>`
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/b1e78356-553b-46a3-87c2-1fc553a9ddc2)

## Accessibility Standard
WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Comment by Becapa** [comment]
I spent some time with a fresh Drupal instance to see if this was a Drupal core issue.  From what I can tell is this is not an issue in core. I'm not able to quite pin down what's causing it, but I think it might have something to do with custom code that is moving our descriptions around (usually to go beneath the label). In the case of the textarea this ends up moving the description one more div layer inside the element which changes its parents which in-turn changes its ID. Then the 'aria-describedby' on the grandparent no longer points to the correct ID.

@qjac
